docs/data-ai/data/data-pipelines.md

```diff
@@ -0,0 +1,756 @@
+            "total": bson.M{"$sum": 1},
+        },
+    }),
+}
+
+_, err := dataClient.UpdateDataPipeline(context.Background(), &datapb.UpdateDataPipelineRequest{
+    Id: "<pipeline-id>",
+    Name: "updated-name",
+    MqlBinary: pipeline,
+    Schedule: "0 * * * *",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.UpdateDataPipeline`](/dev/reference/apis/data-client/#updatedatapipeline) to alter an existing data pipeline:
+
+```typescript
+const apiKey = "<api-key>";
+const apiKeyID = "<api-key-id>";
+
+const client = await createViamClient({
+  credential: {
+    type: "api-key",
+    payload: { key: apiKey, keyId: apiKeyID },
+  },
+});
+
+const dataClient = client.dataClient;
+
+const pipeline = [
+  BSON.serialize({ $match: { component_name: "sensor" } }),
+  BSON.serialize({
+    $group: {
+      _id: "$part_id",
+      total: { $sum: 1 },
+    },
+  }),
+];
+
+await dataClient.updateDataPipeline({
+  id: "<pipeline-id>",
+  name: "updated-name",
+  mqlBinary: pipeline,
+  schedule: "0 0 * * *", // Change to daily
+});
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Enable a pipeline
+
+{{< tabs >}}
+{{% tab name="CLI" %}}
+
+Use [`datapipelines enable`](/dev/tools/cli/#datapipelines) to enable a disabled data pipeline:
+
+```sh {class="command-line" data-prompt="$"}
+viam datapipelines enable --id=<pipeline-id>
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+Use [`DataClient.EnableDataPipeline`](/dev/reference/apis/data-client/#enabledatapipeline) to enable a disabled data pipeline:
+
+```python
+data_client = DataClient.from_api_key(
+    api_key="<api-key>",
+    api_key_id="<api-key-id>"
+)
+
+data_client.enable_data_pipeline(id="<pipeline-id>")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+Use [`DataClient.EnableDataPipeline`](https://pkg.go.dev/go.viam.com/rdk/app#DataClient.EnableDataPipeline) to enable a disabled data pipeline:
+
+```go
+client, err := utils.NewViamClient(context.Background(), utils.WithAPIKey("<api-key>", "<api-key-id>"))
+if err != nil {
+    panic(err)
+}
+defer client.Close()
+
+dataClient := client.DataClient()
+
+_, err := dataClient.EnableDataPipeline(context.Background(), &datapb.EnableDataPipelineRequest{
+    Id: "<pipeline-id>",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.EnableDataPipeline`](/dev/reference/apis/data-client/#enabledatapipeline) to enable a disabled data pipeline:
+
+```typescript
+const apiKey = '<api-key>';
+const apiKeyID = '<api-key-id>';
+
+const client = await createViamClient({
+  credential: {
+    type: 'api-key',
+    payload: { key: apiKey, keyId: apiKeyID }
+  }
+});
+
+const dataClient = client.dataClient;
+
+const pipeline = [
+
+await dataClient.enableDataPipeline({ id: "<pipeline-id>" });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Disable a pipeline
+
+Disabling a data pipeline lets you pause data pipeline execution without fully deleting the pipeline configuration from your organization.
```

Comment:
Maybe note that any time windows that pass while the pipeline is disabled will not contain data and will not be backfilled if the pipeline is enabled again

Reply:
Great point, added information about this.